README.md

```diff
@@ -109,6 +115,10 @@ make changes to the messages:
 You can then open a pull request from the `message-updates-yyyymmdd` branch to
 [the l10n repo](https://github.com/mozilla-l10n/fx-private-relay-l10n) `main` branch.
 
+If you're not yet ready to submit some strings for translation, you can
+tentatively add them to frontend/pendingTranslations.ftl. Strings in that file
```

Comment:
What kind of checks are you thinking of? (In the current setup, these only show up where we'd show `???` otherwise, for which I don't think we have any checks?)

Reply:
Just someting to help us not forget to move/put them into the real `.ftl` file?

Reply:
Ah! Yes, that's a good one, added that in 8a315da5.